app/models/validations/sales/financial_validations.rb

```diff
@@ -79,4 +100,26 @@ def is_relationship_child?(relationship)
   def is_economic_status_child?(economic_status)
     economic_status == 9
   end
+
+  EQUITY_RANGES_FOR_TYPES_BY_COLLECTION_START_YEAR = {
+    2022 => {
+      2 => { "min" => 25, "max" => 75 },
+      30 => { "min" => 10, "max" => 75 },
+      18 => { "min" => 25, "max" => 75 },
+      16 => { "min" => 10, "max" => 75 },
+      24 => { "min" => 25, "max" => 75 },
+      # 28 => ranges don't apply
+      31 => { "min" => 0, "max" => 75 },
+    },
+    2023 => {
+      2 => { "min" => 10, "max" => 75 },
+      30 => { "min" => 25, "max" => 75 },
+      18 => { "min" => 25, "max" => 75 },
+      16 => { "min" => 10, "max" => 75 },
+      24 => { "min" => 25, "max" => 75 },
+      # 28 => ranges don't apply
+      31 => { "min" => 0, "max" => 75 },
+      32 => { "min" => 0, "max" => 75 },
```

Comment:
happy to hear suggestions on a better way to represent this, only differences are the mins for type == 2 and 30 and that the 2023 year has type 32 as well as the 2022 types